plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/SteppableCek/Internal.hs

```diff
@@ -328,8 +329,8 @@ cekTrans :: forall uni fun ann s
            . (ThrowableBuiltins uni fun, GivenCekReqs uni fun ann s)
            => CekTrans uni fun ann s
 cekTrans = \case
-    Starting term          -> pure $ Computing NoFrame Env.empty term
-    Computing ctx env term -> computeCek ctx env term
+    Starting term          -> pure $ Computing NoFrame Env.empty (forgetAnn term)
+    Computing ctx env term -> computeCek ctx env (rememberAnn term)
```

Comment:
This is only safe if `Computing ...` is constructed by `cekTrans` which is not ideal. Probably, it is possible to have some witness value to allow this conditionally